engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/analyse/alias/Graph.scala

```diff
@@ -85,6 +105,12 @@ sealed class Graph extends Serializable {
   ): Option[Graph.Link] = {
     scopeFor(occurrence.id).flatMap(_.resolveUsage(occurrence).map { link =>
       links += link
+      sourceLinks = sourceLinks.updatedWith(link.source)(v =>
```

Comment:
Looks like the same code as in `initLinks`. Maybe really rename `initLinks` to `addLinks` and call it here?